VideoPlayer/Video.js

```diff
@@ -253,6 +265,7 @@ const VideoBase = class extends Component {
 						controls={false}
 						key={sourceKey}
 						mediaComponent={mediaComponent}
+						playsInline
```

Comment:
this is always set to true. should it be like this?

Reply:
Thank you for this observation. I took the liberty of not making it available to changing. So "playsInline" will always be true. I refactored accordingly.